Simplenote/Classes/NSString+Simplenote.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+
+// MARK: - NSString Simplenote Helpers
+//
+extension NSString {
+
+    /// Returns the full range of the receiver
+    ///
+    @objc
+    var rangeOfEntireString: NSRange {
```

Comment:
`fullRange` to be a slightly more concise? `self.fullRange` vs `self.rangeOfEntireString`. Thoughts?

Reply:
Sold!! Updating, thank you Eric!!